proxies/shared/resources/jsc/SetResponseDefaults.js

```diff
@@ -17,3 +17,15 @@ context.setVariable("error.header.Cache-Control", "no-cache, no-store, must-reva
 // set our x-content-type-options-header
 context.setVariable("response.header.X-Content-Type-Options", "nosniff");
 context.setVariable("error.header.X-Content-Type-Options", "nosniff");
+
+// format errors and response objects
+try {
+  const errorContent = context.getVariable("error.content")
+  const responseContent = context.getVariable("response.content")
+
+  if (errorContent) context.setVariable("error.content", JSON.stringify(JSON.parse(errorContent)))
```

Comment:
Does this work without `{` and `}`? I didn't expect rhinojs to be so forgiving for some reason

Reply:
Surprisingly it was ok